src/test/java/org/commcare/formplayer/tests/CaseClaimTests.java

```diff
@@ -57,6 +57,66 @@ protected String getMockRestoreFileName() {
         return "restores/caseclaim.xml";
     }
 
+    @Test
+    public void testDefaultsOverride() throws Exception {
+        configureQueryMock();
+
+        // 1. No user inputs
+        Hashtable<String, String> inputs = new Hashtable<>();
+        QueryData queryData = new QueryData();
+        queryData.setInputs("search_command.m1", inputs);
+        queryData.setExecute("search_command.m1", true);
+        sessionNavigateWithQuery(new String[]{"1", "action 1"},
+                "caseclaim",
+                queryData,
+                true,
+                EntityListResponse.class);
+
+
+        // 2. Override default with an input not in defaults
+        inputs.put("case_type", "case4");
+        sessionNavigateWithQuery(new String[]{"1", "action 1"},
+                "caseclaim",
+                queryData,
+                true,
+                EntityListResponse.class);
+
+        // 3. Override default with an input that is present in defaults
+        inputs.put("case_type", "case2");
+        sessionNavigateWithQuery(new String[]{"1", "action 1"},
+                "caseclaim",
+                queryData,
+                true,
+                EntityListResponse.class);
+
+        // 4. Override default with multiple inputs one valid and other invalid
+        inputs.put("case_type", "case2#,#case4");
+        sessionNavigateWithQuery(new String[]{"1", "action 1"},
+                "caseclaim",
+                queryData,
+                true,
+                EntityListResponse.class);
+
+        // 5. Add an user input not present in defaults
```

Comment:
same test description as 2 - do you mean "Add a user input for a param that has no defaults"?

Reply:
yes, updated description